integration-libs/epd-visualization/_index.scss

```diff
@@ -1,10 +1,9 @@
-@import '~@spartacus/styles/scss/core';
-@import '~@spartacus/styles/scss/app';
```

Comment:
Why did app get removed. Was there no instances of some re-used classes from our styles?

Reply:
Same applies for cart and order. Have to make sure we didn't miss anything